src/content/en/2021/css.md

```diff
@@ -525,9 +525,9 @@ In color formats where the value has historically used commas inside a functiona
    <thead>
     <tr>
       <td></td>
```

Comment:
Should this be `th`?

Reply:
Nope.

If it was this wouldn't be an issue. Yeaah!
However then PrinceXML complains about an empty heading. Booo

The correct, accessible markup an empty heading cell is apparently `<td></td>` Which looks weird, but that's what you supposed to do apparenlty! Remember researching this when the ebook first launched as had to set a few of them to that.

The only problem is that automatically turns it into a complex table in PrinceXML's eyes, hence need to explicitly define the roles for the `<th>`'s.